en/simulation/gazebo.md

```diff
@@ -375,6 +375,11 @@ The code is available on the [sitl_gazebo repository](https://github.com/px4/sit
 > **Note** The build system enforces the correct GIT submodules, including the simulator.
   It will not overwrite changes in files in the directory.
 
+In case something is not working as expected, you can increase the verbosity of the gazebo simulation through `VERBOSE_SIM`
```

Comment:
Should we specify if this is to `gzserver` or `gzclient` or both?

Reply:
@Jaeyoung-Lim 
Thanks. I have moved to its own heading/section. This isn't a bad place at all - I just think it is useful to have all the info about flags in one place, and I think a search is more likely to find this. 

FWIW it this only logs the server or the client we should say that (ie I agree with Nuno)